ProcessLib/Utils/GlobalVectorUtils.h

```diff
@@ -0,0 +1,53 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
```

Comment:
Maybe this file can be merged with `NumLib/DOF/DOFTableUtil.h`, because there already are functions accessing nodal values, computing norms etc.